alicloud/resource_alicloud_slb_listener.go

```diff
@@ -82,6 +82,24 @@ func resourceAliyunSlbListener() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+
+			"acl_status": &schema.Schema{
+				Type:         schema.TypeString,
+				ValidateFunc: validateAllowedStringValue([]string{string(OnFlag), string(OffFlag)}),
+				Optional:     true,
+			},
+
+			"acl_type": &schema.Schema{
+				Type:         schema.TypeString,
+				ValidateFunc: validateAllowedStringValue([]string{string(AclTypeBlack), string(AclTypeWhite)}),
+				Optional:     true,
+			},
+
+			"acl_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
```

Comment:
If you want to modify the existing resource `alicloud_slb_listener`, please use a new pull-request to do it. This PR is used to add new resource acl